portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,154 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <div id = "banner">
+        <h1>Hello World!</h1>
+        <div id="banner-content">Cirque de Towers, Wyoming</div>
+    </div>    
+
+    <div id = "profile-pic">
+        <img style = "width: 200px; height = 200px" src="/images/siteimages/climbing.jpg">
+    </div>
+
+    <div id = "side-pic">
+        <img src = "/images/siteimages/truck.jpg" style = "width: 250px; height = 250px;">
+    </div>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p style = "padding: 20px">
+          My name is Meleah, but I go by Mel. I love hiking, climbing, and working 
+          on projects. I am studying Computer Engineering, but find myself just as
+          excited about learning to code! I love being outdoors and exploring the 
+          mountains. Maybe it's a side effect of growing up in the flat state of 
+          Texas, but going to the mountains always brings me so much joy. I plan to
+          attempt the Seven Summits one day. I also enjoy working on old cars and have
+          been fixing up my truck in my free time.
+      </p>
+    </div>
+
+<!-- I originally had this as div row/cols but I was struggling to get the 
```

Comment:
Nice job layout the page and finding the workaround!

Usually we still do this by CSS. A [table layout is consider a bad practice](https://teamtreehouse.com/community/why-is-it-considered-bad-practice-to-use-tables-markup-a-website).

You don't need to change it now, but a suggestion is that when thinking about building a website, try to make the HTML describing the content instead of the style. I.e. if it's a paragraph, use `<p>`, if it's a header, use `<h1>`, if it's a table, use `<table>`. And in CSS, you try to give those element styles you want.

The has the following benefit:

- better SEO
- better rendering on all different devices / browsers.

Reply:
I took a look at the link provided and understand the best practice moving forward